pkg/clients/aws/ecr.go

```diff
@@ -0,0 +1,37 @@
+package aws
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/service/ecr"
+)
+
+type DockerCredentials struct {
```

Comment:
nit: we could move this to `types` to avoid having to import this package from the packages that consume this as an interface